quickwit/quickwit-integration-tests/src/tests/update_tests/doc_mapping_tests.rs

```diff
@@ -220,9 +220,9 @@ async fn test_update_doc_mapping_json_to_text() {
         ingest_after_update,
         &[
             ("body:hello", Ok(&[json!({"body": "hello world"})])),
-            // error expected because the validation is performed
-            // by latest doc mapping
-            ("body.field1:hello", Err(())),
+            // ok because we can query for non existent field and they are ignored, but we don't
+            // get the field because it was deleted. This spec is very odd :-/
```

Comment:
I agree this behavior is very weird! Where is this spec described? In this test, we use the dynamic (default) mode. Shouldn´t we get all the fields when that's the case?

Reply:
ho, i thought the tests where on a lenient doc mapping, but no, it's just the absence of #5281 showing itself

Reply:
ah, so we should definitively merge that and this should work!

Reply:
hum, nevermind, body didn't disappear, it was transformed from json to text, which we currently make return nothing (there is no good way to put a complex object in a string)